File: java/src/test/java/api/arch/CorpCoopDemoTest.java
package api.arch;

import java.util.Map;

public class CorpCoopDemoTest {
    public static void main(String[] args) throws Exception {
        CorpCoopDemo corpCoopDemo = new CorpCoopDemo();
        // 列出我连接的企业
        corpCoopDepartList(corpCoopDemo);
        // 列出我连接的企业对接人
        corpCoopMemberList(corpCoopDemo);
        // 列出我连接的企业对接人详细信息
        corpCoopUserInfo(corpCoopDemo);
    }

    private static void corpCoopDepartList(CorpCoopDemo corpCoopDemo) throws Exception {
        Map<String, Object> result = corpCoopDemo.corpCoopDepartList(null);
        System.out.println("deptList result\n" + result);
    }

    private static void corpCoopMemberList(CorpCoopDemo corpCoopDemo) throws Exception {
        Map<String, Object> result = corpCoopDemo.corpCoopMemberList(null);
        System.out.println("corpCoopMemberList result\n" + result);
    }

    private static void corpCoopUserInfo(CorpCoopDemo corpCoopDemo) throws Exception {
        Map<String, Object> result = corpCoopDemo.corpCoopUserInfo("R-61150d0d497d9f0006448073-jdy-pr55od7jrsl7");
        System.out.println("corpCoopUserInfo result\n" + result);
    }
}

File: python/test/api/arch/corp_coop_demo_test.py
from src.api.arch.corp_coop_demo import CorpCoopApi
from src.constants.http_constant import HttpConstant

corpCoopApi = CorpCoopApi(HttpConstant.API_KEY,HttpConstant.HOST)


#  列出我连接的企业
def corpCoopDepartList():
    result = corpCoopApi.corpCoopDepartList(None, None)
    print('corpCoopDepartList result:', result)


#  列出我连接的企业对接人
def corpCoopMemberList():
    result = corpCoopApi.corpCoopMemberList(None, 'v4')
    print('corpCoopMemberList result:', result)


# 列出我连接的企业对接人详细信息
def corpCoopUserInfo():
    result = corpCoopApi.corpCoopUserInfo('R-60e2767055c8760006ac79bc-jdy-y4r83c4jpqzg', 'v2')
    print('corpCoopUserInfo result:', result)


if __name__ == '__main__':
    corpCoopDepartList()
    corpCoopMemberList()
    corpCoopUserInfo()
